index.src.html

```diff
@@ -572,12 +583,25 @@ <h4 id="clear-cache">
     2.  Let |cache list| be the set of entries from the <a>network cache</a> whose `target URI`
         [=url/host=] is identical to |host|.
 
-    3.  For each |entry| in |cache list|:
+        1.  For each |entry| in |cache list|:
    
-        1.  Remove |entry| from the <a>network cache</a>.
+            1.  Remove |entry| from the <a>network cache</a>.
+
+        2.  If a user agent implements caches beyond a pure <a>network cache</a>, it MUST remove
+            all entries from those caches which match |origin|.
+
+    3.  For each |traversable| in the user agent's set of [=top-level traversable=]:
+
+        1.  For each |entry| in the |traversable|'s [=session history entries=]:
+
+            1.  Let |state| be |entry|'s <a>document state</a> whose `origin` attribute is identical
+                to |host|.
+
+            2.  Let |document| be |state|'s `document` attribute.
```

Comment:
There is no such thing as a "`document` attribute". I think you want the document state's [=document state/document=].

Reply:
I'm a bit confused by the autolink, seems [=document state/document=] couldn't find the right dfn. I added an explicit dfn entry for it.

Reply:
Yeah, it's not exported; thus my comment about reaching into the internals in this way.